src/autotype/AutoTypeMatchView.h

```diff
@@ -35,7 +35,7 @@ class AutoTypeMatchView : public QTableView
     explicit AutoTypeMatchView(QWidget* parent = nullptr);
     AutoTypeMatch currentMatch();
     AutoTypeMatch matchFromIndex(const QModelIndex& index);
-    void setMatchList(const QList<AutoTypeMatch>& matches, bool selectFirst);
+    void setMatchList(const QList<AutoTypeMatch>& matches, int selectedIndex, bool selectFirst);
```

Comment:
`selectFirst` looks redundant with `selectedIndex`.

Reply:
It's not actually redundant. `selectedIndex` is an index to the match list and not an index to the sorted view. I need to be able to distinguish from selecting the first _visible_ item in the list and the first item in the input list. If it can be easily compressed into a single argument that makes sense I'll gladly refactor it.

Reply:
Then maybe just a rename to clarify the difference? Something like `matchIndex` or `inputIndex` or `selectedMatch` for `selectedIndex`, and `selectFirstVisible` for `selectFirst`.

Reply:
We discussed this on Matrix with @droidmonkey and decided to split this into three distinct functions where `setMatchList` just clears the selection so the call isn't too overloaded.

Reply:
I've now split it into separate functions.